### PR TITLE
Poloidal degree change

### PR DESCRIPTION
**Pull request by FusionSandwich**
@connoramoreno please take a look and let us know what you think.

### CODE REVIEW

File: parastell/nwl_utils.py [hunk 1]
```diff
@@ -186,6 +186,11 @@ def flux_coords(plas_eq, wall_s, coords, num_threads):
             for theta_coord_chunk in theta_coord_chunks
             for theta_coord in theta_coord_chunk
         ]
+    # Ensures theta_coords are all positive (add 360 degrees where needed)
+    theta_coords = [
+        (theta if theta >= 0 else theta + 2 * np.pi) for theta in theta_coords
+    ]
```
Comment (connoramoreno) [inline parastell/nwl_utils.py:192]:
```suggestion
    theta_coords = (theta_coords + 2 * np.pi) % 2*np.pi
```
Avoids the use of conditionals

**Review by connoramoreno — CHANGES_REQUESTED** [review]
Just a small suggestion for the computation of positive angles; otherwise LGTM!

File: parastell/nwl_utils.py (hunk 1 above)
Comment (connoramoreno) [reply parastell/nwl_utils.py:192]: You should be able to directly commit this suggestion

Comment (FusionSandwich) [reply parastell/nwl_utils.py:192]: That's pretty slick!

**Comment by connoramoreno** [comment]
I can't merge until this is formatted with black

**Comment by connoramoreno** [comment]
My suggestion must've done something to the formatting

File: parastell/nwl_utils.py [hunk 2]
```diff
@@ -186,6 +186,9 @@ def flux_coords(plas_eq, wall_s, coords, num_threads):
             for theta_coord_chunk in theta_coord_chunks
             for theta_coord in theta_coord_chunk
         ]
+    # Ensures theta_coords are all positive (add 360 degrees where needed)
+    theta_coords = (theta_coords + 2 * np.pi) % 2*np.pi
```
Comment (gonuke) [inline parastell/nwl_utils.py:190]: will they need to be reordered/sorted?

Comment (Edgar-21) [reply parastell/nwl_utils.py:190]: Doesn't need to be reordered, theta_coords is just a list of the same length as the list of surface crossings from the surface source that stores the poloidal angle for each surface crossing, all we are doing here is changing the notation of theta, each theta_coord still corresponds to the same surface crossing.

For reference, here is what the NWL looks like from before:
image
And after:
image

The first has been rescaled as noted in its title, but looking at the overall profile the coordinate shift can be seen

Comment (gonuke) [inline parastell/nwl_utils.py:190]: I think this is why black is complaining.
```suggestion
    theta_coords = (theta_coords + 2 * np.pi) % 2 * np.pi
```

Comment (connoramoreno) [reply parastell/nwl_utils.py]: Thanks!

**Review by connoramoreno — APPROVED** [review]
Looks good, thanks @FusionSandwich

**Pull request merged**